mpas_analysis/ocean/meridional_overturning_circulation.py

```diff
@@ -224,6 +242,42 @@ def _load_mesh(runStreams):  # {{{
         refTopDepth, refLayerThickness  # }}}
 
 
+def _cache_velocity_climatologies(config, startDateClimo, endDateClimo,
+                                  inputFilesClimo, simulationStartTime,
+                                  variableMap, calendar):  # {{{
+    '''compute yearly velocity climatologies and cache them'''
+
+    variableList = ['avgNormalVelocity',
+                    'avgVertVelocityTop']
+
+    outputDirectory = build_config_full_path(config, 'output',
+                                             'mpasClimatologySubdirectory')
+
+    try:
+        os.makedirs(outputDirectory)
+    except OSError:
```

Comment:
Maybe comment here that we don't care about the error raised if the directory already exists.  This code is a little dangerous, however, because it doesn't catch errors that could be a problem, e.g., what happens if you try to write to a full or write-protected location?